lib/neuron/config.ex

```diff
@@ -25,7 +25,8 @@ defmodule Neuron.Config do
 
   - url: graphql endpoint url
   - headers: headers to be sent in the request
-  - connection_opts: additional options to be passed to HTTPoison
+  - connection_opts: additional options to be passed to connection library
+  - connection_library:
```

Comment:
I would rather call this `connection_module`, because it's not necessarily some external library

Reply:
sure 👌